crates/esplora/Cargo.toml

```diff
@@ -13,16 +13,16 @@ readme = "README.md"
 
 [dependencies]
 bdk_chain = { path = "../chain", version = "0.9.0", default-features = false }
-esplora-client = { version = "0.6.0", default-features = false }
+esplora-client = { version = "0.7.0", default-features = false, path = "../../../../rust-esplora-client/02-07-upgrade-bitcoin-0.31" }
```

Comment:
The `rust-esplora-client` 0.7.0 is now published on crates.io so this can be fixed.

Reply:
Thanks @notmandatory, will do.